Mecrano/Personal-Page
# Feat/change header desing

#### What problem is this solving?

Change Header Style improving UI/UX

#### How to test it?

1. execute yarn dev
2. See Changes

#### Screenshots or example usage:

![image](https://user-images.githubusercontent.com/26680887/187501936-e43bbac8-2842-449a-ac62-876e9afe4074.png)
![image](https://user-images.githubusercontent.com/26680887/187502046-b7e7ebf3-1a95-4cdf-88b1-462e00b44264.png)
![image](https://user-images.githubusercontent.com/26680887/187502099-3f0a8686-276e-4203-ade7-728c9494ae46.png)


#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/muHlnG1WU9YfS/giphy.gif)
